src/jaguar.toit

```diff
@@ -215,11 +202,58 @@ flash-image image-size/int reader/reader.Reader name/string? defines/Map --crc32
     return image
   unreachable
 
-run-image image/uuid.Uuid cause/string name/string? defines/Map -> containers.Container:
+/**
+Callbacks that are scheduled to run at a specific time.
+This is used to kill containers that have deadlines.
+*/
+scheduled-callbacks := ScheduledCallbacks
+
+run-image image/uuid.Uuid cause/string name/string? defines/Map -> none:
+ network-disabled := (defines.get JAG-NETWORK-DISABLED) == true
```

Comment:
The indentation here is just one, but that's because the "task::" below will disappear in a follow-up PR, and this way there are fewer changes.